src/storm/modelchecker/blackbox/bound-functions/boundFunc.h

```diff
@@ -1,16 +1,21 @@
-//
-// Created by Maximilian Kamps on 25.05.23.
-//
-
-#ifndef STORM_BOUNDFUNC_H
-#define STORM_BOUNDFUNC_H
+#pragma once
 
 #include <utility>
 
+
+/*!
+ * Abstract Base Class for the function INTERVAL
+ */
 template <typename ValueType>
-class BoundFunc {
+class boundFunc {
    public:
-     virtual std::pair<ValueType,ValueType> INTERVAL(int totalSamples, int partialSample, double delta) = 0; //abstract method
+    /*!
+     * Calculates the lower and upper bound for a transition for the eMDP
+     * @param totalSamples : total Samples for one Action generated in Simulate
+     * @param partialSample : Samples for (state,transition,state) generated in Simulate
+     * @param delta : Inconfidence value delta
+     * @return : upper and lower bound
+     */
+    virtual std::pair<ValueType,ValueType> INTERVAL(int totalSamples, int partialSample, double delta) = 0; //abstract method
```

Comment:
Ist integer als Typ für den SampleCount sinnvoll oder sollte ein größerer Datentyp z.B uint64 verwendete werden?
Die Frage betrifft auch den eMDP, daher sollten wir das zusammen in der Runde nochmal besprechen